nbodykit/algorithms/pair_counters/base.py

```diff
@@ -47,7 +47,14 @@ def __init__(self, mode, edges, first, second, Nmu, pimax, show_progress=False):
         # store the total size of the sources
         self.attrs['N1'] = first.csize
         self.attrs['N2'] = second.csize if second is not None else None
-
+        
+       	if second is None or second==first:
```

Comment:
Looks like a tab/space mixture issue -- we only use 4 spaces for indentation, no tabs.

Reply:
Also to test if two fields we can probably use 'is' instead of ==.